src/hooks/useSwapCurrencyHandlers.js

```diff
@@ -123,42 +123,42 @@ export default function useSwapCurrencyHandlers({
   ]);
 
   const flipSwapCurrenciesWithTimeout = useCallback(
-    (outputIndependentField = false, independentValue = null) =>
-      setTimeout(
-        () =>
-          dispatch(
-            flipSwapCurrencies(outputIndependentField, independentValue)
-          ),
-        50
-      ),
+    (focusToRef, outputIndependentField = false, independentValue = null) => {
+      InteractionManager.runAfterInteractions(() => {
+        dispatch(flipSwapCurrencies(outputIndependentField, independentValue));
+        setTimeout(() => {
+          focusTextInput(focusToRef.current);
+        }, 50);
+      });
+    },
     [dispatch]
   );
 
   const flipCurrencies = useCallback(() => {
     if (currentNetwork === Network.arbitrum) {
       outputFieldRef.current?.clear();
-      focusTextInput(inputFieldRef.current);
+      nativeFieldRef.current?.focus?.();
```

Comment:
why are we focusing 2 inputs here?

Reply:
@skylarbarrera it was the only way to refocus into the input again for arbitrum, since we don't focus to the output here